dl_controller:
  num_goal_sets: 1
  goal_features1: [298.0,214.0,315.0,224.0,400.0,276.0,414.0,292.0]
